spark/v3.3/spark-extensions/src/test/java/org/apache/iceberg/spark/extensions/TestRewritePositionDeleteFiles.java

```diff
@@ -221,7 +221,7 @@ private <T> void testDanglingDelete(int numDataFiles) throws Exception {
             .execute();
 
     List<DeleteFile> newDeleteFiles = deleteFiles(table);
-    Assert.assertEquals("Should have removed all dangling delete files", 0, newDeleteFiles.size());
+    assertThat(newDeleteFiles.size()).as("Remaining dangling deletes").isEqualTo(0);
```

Comment:
```suggestion
    assertThat(newDeleteFiles).as("Remaining dangling deletes").isEmpty();
```
the advantage of having the assertion like that is that it will show the content of `newDeleteFiles` when the assertion fails, making it easier to debug